test/controllers/restaurants_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class RestaurantsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @restaurant = restaurants(:one)
+  end
+
+  test "should get index" do
+    get restaurants_url
+    assert_response :success
```

Comment:
while this works, we use `rspec` for testing, i think it's worth knowing at least the basics